lib/EnsEMBL/REST/RegistryHelper.pm

```diff
@@ -137,7 +137,6 @@ sub build_species_info {
         $processed_db{$db_key} = 1;
       }
     }
-    $dba->dbc->db_handle->disconnect;

```

Comment:
I'm a bit wary of this, but I realise you're managing the connections in another way.

Reply:
I added that line originally to fix an situation that no longer arises - it should not have made it into the previous PR